fdbclient/include/fdbclient/RunRYWTransaction.actor.h

```diff
@@ -53,6 +53,21 @@ Future<decltype(std::declval<Function>()(Reference<ReadYourWritesTransaction>())
 	}
 }
 
+ACTOR template <class Function>
+Future<Void> runRYWTransactionVoid(Database cx, Function func) {
+	state Reference<ReadYourWritesTransaction> tr(new ReadYourWritesTransaction(cx));
+	loop {
+		try {
+			// func should be idempotent; otherwise, retry will get undefined result
```

Comment:
What do you mean by retry getting undefined result? Is it that it will cause unintended effects?

Reply:
This is just a copy of the function above but for functions that return `Future<Void>`, so the comment is the same as above. My guess, though, is that this is saying that your function could be run multiple times, and if it isn't idempotent then the result could vary based on how many times it is run.